config/default.yml

```diff
@@ -38,8 +38,32 @@ Style/EmptyElse:
 Style/Documentation:
   Enabled: no
 
+Naming/MemoizedInstanceVariableName:
+  EnforcedStyleForLeadingUnderscores: optional
+
+Metrics/MethodLength:
+  Max: 15
+
+Metrics/BlockLength:
+  Enabled: false
```

Comment:
Enforced by MethodLength inside methods, and doesn't make much sense outside methods (routes.rb, value objects...)